pkg/plugins/k8s/k8s_plugin.go

```diff
@@ -171,18 +187,24 @@ func (p *K8sPlugin) readSwitchdevManifest() error {
 		Name:    "ConditionPathExists",
 		Value:   "!/etc/ignition-machine-config-encapsulated.json",
 	}
-	switchdevService, err = service.RemoveFromService(switchdevService, conditionOpt)
+	switchdevBeforeNMService, err = service.RemoveFromService(switchdevBeforeNMService, conditionOpt)
```

Comment:
I can see this condition in switchdevAfterNMService as well

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/c400742d910d731dfda9e81475d3b8047f051485/bindata/manifests/switchdev-config/switchdev-units/switchdev-configuration-after-NM.yaml#L5

I believe it should be removed as well.

Reply:
Done

Reply:
Done